src/GraphQL.Tests/DI.xUnit/DependencyInjectionDataAttribute.cs

```diff
@@ -0,0 +1,8 @@
+namespace GraphQL.Tests.DI;
+
+internal sealed class DependencyInjectionDataAttribute : ClassDataAttribute
```

Comment:
There are no usages of this attribute in tests now by I tested it. It works. This attribute should be used when someone adds one more DI provider.

Reply:
i can add another di provider

Reply:
I'm experimenting with StructureMap` now. Some variants:
1. Autofac
2. LightInject
3. Ninject
4. SimpleInjector
5. StructureMap
6. CastleWindsor

Rel: https://github.com/EasyNetQ/EasyNetQ/pull/1108